pkg/installer/install

```diff
@@ -391,9 +391,13 @@ RAID_LEVEL="none"
 eve_flavor=$(cat /root/etc/eve-hv-type)
 
 if [ "$eve_flavor" = "k" ]; then
-   INSTALL_CLUSTERED_STORAGE=true
-   INSTALL_ZFS=true
-   logmsg "Kubevirt image installing ZFS"
+   logmsg "cmdline: $(cat /proc/cmdline)"
+   if [ "$INSTALL_ZFS" = "true" ]; then
+      INSTALL_CLUSTERED_STORAGE=true
```

Comment:
Does this change mean that a user who wants clustered_storage need to explicitly install with ZFS enabled?